opencensus/plugins/grpc/internal/grpc_plugin.cc

```diff
@@ -52,6 +55,11 @@ void RegisterGrpcPlugin() {
   RpcServerResponseCount();
 }
 
+opencensus::trace::Span GetSpanFromServerContext(grpc::ServerContext* context) {
```

Comment:
We may want to expand this to allow for GetSpanFromClientContext as well.  This will require additional work though as the ClientContext initially just has a nullptr for the census_context.  We would need to setup functionality for populating it (this already exists for google3).  This works for now.